react/__tests__/index.test.tsx

```diff
@@ -155,5 +160,57 @@ describe('GA4 events', () => {
         promotion_name: 'Summer Sale',
       })
     })
+
+    it('sends an event that signifies an item being added to the cart', () => {
+      type CartItemMockType = Pick<
+        CartItem,
+        | 'name'
+        | 'brand'
+        | 'price'
+        | 'skuId'
+        | 'skuName'
+        | 'quantity'
+        | 'category'
+        | 'productId'
+      >
+
+      const cartItem: CartItemMockType = {
+        productId: '200000202',
+        skuId: '2000304',
+        brand: 'Sony',
+        name: 'Top Wood',
+        skuName: 'top_wood_200',
+        price: 197.99,
+        category: 'Home & Decor',
+        quantity: 1,
+      }
+
+      const data: AddToCartData = {
+        currency: 'USD',
+        event: 'addToCart',
+        eventName: 'vtex:addToCart',
+        items: [cartItem as CartItem],
```

Comment:
Assuming there could be multiple items being added to the cart, as I mentioned in a previous comment, could you increment this test to have 2 items? Just to be testing a more thorough case.